### PR TITLE
Fix StepperRow layout when imageView is set

### PR DESCRIPTION
**Pull request by nidegen**
Addressing #1838 

I ended up copying the code from SliderRow and adapting it slightly.

One Remark:  I prefer the value label text color  to be grey like in the SliderRow, but I have left it as it was for now.

### CODE REVIEW

File: Source/Rows/StepperRow.swift [hunk 1]
```diff
@@ -8,88 +8,154 @@
...
+    let vContraint = NSLayoutConstraint(item: stepper!, attribute: .centerY, relatedBy: .equal, toItem: contentView, attribute: .centerY, multiplier: 1, constant: 0)
+    dynamicConstraints.append(vContraint)
+    contentView.addConstraints(dynamicConstraints)
+  }
+  
 }
 
-// MARK: StepperRow
 
-open class _StepperRow: Row<StepperCell> {
```
Comment (mats-claassen) [inline Source/Rows/StepperRow.swift]: You should leave an open superclass to allow overrides

File: Source/Rows/StepperRow.swift [hunk 2]
```diff
@@ -8,88 +8,154 @@
...
+      self.titleLabel = title
+      
+      let value = detailTextLabel
+      value?.translatesAutoresizingMaskIntoConstraints = false
+      value?.setContentHuggingPriority(UILayoutPriority(500), for: .horizontal)
+      value?.adjustsFontSizeToFitWidth = true
+      value?.minimumScaleFactor = 0.5
+      self.valueLabel = value
+      let font = self.valueLabel.font!
+      self.valueLabel.font = UIFont.monospacedDigitSystemFont(ofSize: font.pointSize, weight: font.weight)
```
Comment (mats-claassen) [inline Source/Rows/StepperRow.swift]: I don't think it is necessary to set this font here, and if then prefer `systemFont(ofSize:)`.

File: Source/Rows/StepperRow.swift [hunk 3]
```diff
@@ -8,88 +8,154 @@
...
-        row.updateCell()
+    selectionStyle = .none
+    stepper.minimumValue = 0
+    stepper.maximumValue = 10
+    stepper.addTarget(self, action: #selector(StepperCell.valueChanged), for: .valueChanged)
+  }
+  
+  open override func update() {
+    super.update()
+    titleLabel.text = row.title
```
Comment (mats-claassen) [inline Source/Rows/StepperRow.swift]: This is done in `super.update`

File: Source/Rows/StepperRow.swift (hunk 2 above)
Comment (nidegen) [reply Source/Rows/StepperRow.swift]: ok sure. As i was copying from SliderRow first, the numbers were positioned after the stepper. There, it shifted on ever digit change. There the monospaced digit font was a great fit.

File: Source/Rows/StepperRow.swift (hunk 1 above)
Comment (nidegen) [reply Source/Rows/StepperRow.swift]: sure, that comes from SliderView - ok if I fix it there too then?

**Comment by nidegen** [comment]
OK, I adressed everything and cleaned up the indentation. Didn't touch SliderRow though. ptal

File: Source/Rows/StepperRow.swift (hunk 1 above)
Comment (mats-claassen) [reply Source/Rows/StepperRow.swift]: Yes

**Pull request merged**